build.zig.zon

```diff
@@ -0,0 +1,16 @@
+.{
+    .name = "zig-curl",
+    .version = "0.0.1",
+    .paths = .{
+        "src",
+        "build.zig",
+        "build.zig.zon",
+        "LICENSE",
+    },
+    .dependencies = .{
+        .libcurl = .{
+            .url = "https://github.com/Cloudef/zigcurl/archive/e9e726131d3eb80c02d20b7ee5ca6935324d6697.tar.gz",
```

Comment:
Which libcurl version does this bundle with?

Reply:
curl 8.1.1
mbedtls 3.4.0
zlib 1.2.13
https://github.com/star-tek-mb/zigcurl

Reply:
Maybe we need to add this in README.

Reply:
Another option could be to merge the work from that repo to this, that way you wouldn't have to depend on anything. I'm pretty sure the author's fine with it.